pages/portfolio.html

```diff
@@ -4,9 +4,41 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <link rel="stylesheet" href="../styles/common.css">
+  <link rel="stylesheet" href="../styles/portfolio.css">
+  <title>Portfolio</title>
 </head>
-<body>
-  
-</body>
+
+<div class="wrapper">
+  <header class="box header">
+    <h1>My Portfolio</h1>
+  </header>
+
+  <nav class="box sidebar">
+    <ul class="no-bullets center">
+      <li><a href="index.html">Home</a></li>
+      <li><a href="about.html">About</a></li>
+      <li><a href="portfolio.html">Portfolio</a></li>
+    </ul>
+  </nav>
+
+  <body class="box content">
+    <section>
```

Comment:
section really wants a heading (an h tag) as a child, so consider uncommenting the h2 you have here.